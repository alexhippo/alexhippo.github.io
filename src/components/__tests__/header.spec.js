import React from "react"
import renderer from "react-test-renderer"
import { render } from "@testing-library/react"

import Header from "../header"

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Header />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("displays alexhipolito title", () => {
    const { getByText } = render(<Header />)
    expect(getByText("alexhipolito")).toBeInTheDocument();
  })

})