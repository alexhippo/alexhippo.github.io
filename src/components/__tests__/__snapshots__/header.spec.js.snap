// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<nav
  className="navbar navbar navbar-expand-lg navbar-light fixed-top"
  id="header"
  role="navigation"
>
  <div
    className="sc-bdfBwQ ieCRGZ"
  >
    <a
      className="nav-link"
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      alexhipolito
    </a>
  </div>
  <button
    aria-controls="basic-navbar-nav"
    aria-label="Toggle navigation"
    className="navbar-toggler collapsed"
    onClick={[Function]}
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    aria-expanded={null}
    className="navbar-collapse collapse"
    id="basic-navbar-nav"
  >
    <div
      className="mr-auto navbar-nav"
      onKeyDown={[Function]}
    >
      <a
        className="nav-link"
        data-rb-event-key={null}
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        style={
          Object {
            "color": "#000",
          }
        }
      >
        about
      </a>
      <a
        className="nav-link"
        data-rb-event-key={null}
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        style={
          Object {
            "color": "#000",
          }
        }
      >
        portfolio
      </a>
      <a
        className="nav-link"
        data-rb-event-key={null}
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        style={
          Object {
            "color": "#000",
          }
        }
      >
        résumé
      </a>
      <a
        className="nav-link"
        data-rb-event-key={null}
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        style={
          Object {
            "color": "#000",
          }
        }
      >
        contact
      </a>
    </div>
  </div>
</nav>
`;
